mediaflow_proxy/extractors/dlhd.py

```
@@ -12,160 +13,258 @@ def __init__(self, request_headers: dict):
+            channel_key = channel_key.group(1) if channel_key else None
+            auth_ts = extract_var(iframe_content, 'c')
+            auth_rnd = extract_var(iframe_content, 'd')
+            auth_sig = extract_var(iframe_content, 'e')
+            auth_host = extract_var(iframe_content, 'a')
+            auth_php = extract_var(iframe_content, 'b')
+            if not all([channel_key, auth_ts, auth_rnd, auth_sig, auth_host, auth_php]):
+                raise ExtractorError("Error extracting parameters: one or more parameters not found")
+            auth_sig = quote_plus(auth_sig)
+            # 6. Richiesta auth
+            auth_url = f'{auth_host}{auth_php}?channel_id={channel_key}&ts={auth_ts}&rnd={auth_rnd}&sig={auth_sig}'
+            auth_resp = await self._make_request(auth_url, headers=daddylive_headers)
```

Ruff (0.12.2)

Local variable `auth_resp` is assigned to but never used

Remove assignment to unused variable `auth_resp`

(F841)